app.py

```diff
@@ -1,13 +1,29 @@
 #!/usr/bin/env python
+import asyncio
+import datetime
 from tornado.web import Application, RequestHandler
 from tornado.ioloop import IOLoop
 from root import RootHandler
 from connections import ConnectionsHandler
+from predict import Predict
+
+async def main():
+    p = Predict()
+    station = "http://example5.com"
```

Comment:
Just to be sure: is this placeholder URI intended or did you forget to replace it?

Reply:
This is just some demo lookup with the fake records in the database. Needs to be replaced when we will actually ise this